src/test/javascript/spec/component/admin/user-management.component.spec.ts

```diff
@@ -37,8 +36,7 @@ describe('UserManagementComponent', () => {
             imports: [ArtemisTestModule, MockModule(ReactiveFormsModule), MockModule(NgbModule)],
             declarations: [
                 UserManagementComponent,
-                MockComponent(AlertErrorComponent),
-                MockComponent(AlertComponent),
+                MockComponent(AlertOverlayComponent),
```

Comment:
Does the component still need to be mocked for the test even if there no longer is a `<jhi-alert-component>`or now a `<jhi-alert-overlay>` in the HTML of the component or could it just be removed?
(Applies to the other tests as well. Potentially makes the comment above redundant.)

Reply:
You're right. I removed the mocks of this component from .. literally everywhere. Tests still pass 👍